authapi/serializers.py

```diff
@@ -51,11 +51,46 @@ class UserSerializer(serializers.ModelSerializer):
         many=True, source='seedteam_set', read_only=True)
     organizations = OrganizationSummarySerializer(
         many=True, source='seedorganization_set', read_only=True)
-    email = serializers.EmailField(source='username')
-    admin = serializers.BooleanField(source='is_superuser')
+    email = serializers.EmailField()
+    admin = serializers.BooleanField(source='is_superuser', required=False)
```

Comment:
The problem with that is, if they are an admin user, and they submit a `PUT` request without the admin field (which is valid because it's not required), then they will turn the user into a normal user, which while technically correct, I thought would be a bad thing to have happen...